query/sources/impl/src/main/java/org/codice/ddf/admin/sources/wfs/WfsSourceUtils.java

```diff
@@ -179,54 +174,20 @@ public WfsSourceUtils(ConfiguratorSuite configuratorSuite) {
                     .evaluate(capabilitiesXml);
         } catch (XPathExpressionException e) {
             LOGGER.debug("Failed to parse XML response.");
-            configResult.addArgumentMessage(unknownEndpointError(responseField.requestUrlField()
+            return Reports.from(unknownEndpointError(responseField.requestUrlField()
                     .path()));
-            return configResult;
         }
 
-        configResult.result(preferredConfig.wfsVersion(wfsVersion));
+        Report<WfsSourceConfigurationField> configResult = Reports.from(preferredConfig.wfsVersion(wfsVersion));
```

Comment:
This is a little out of scope of your work but this logic here isn't making a lot of sense to me. We should be adding the `wfsVersion` to a `WfsVersionField` and only calling validation on it. I think it should look something like:
```
 WfsVersion wfsVersionToCheck = new WfsVersion();
        wfsVersionToCheck.isRequired(true);
        wfsVersionToCheck.setValue(wfsVersion);
        if(!wfsVersionToCheck.validate().isEmpty()) {
            return Reports.from(unknownEndpointError(responseField.requestUrlField()
                    .path()));
        }

        return Reports.from(new WfsSourceConfigurationField()
                .wfsVersion(wfsVersion)
                .endpointUrl(requestUrl.getValue())
                .credentials()
                .username(creds.username())
                .password(FLAG_PASSWORD));
```